src/config.ts

```diff
@@ -80,6 +80,14 @@ export interface Config {
    */
   plugins?: {[pluginName: string]: string;};
 
+  /**
+   * An object that specifies arbitrary options on how to patch a module for
+   * instrumentation. It will be available to plugins through
+   * TraceAgent#getPluginOptions.
+   */
```

Comment:
From this comment I am trying to figure out how users are supposed to use this. Are these options for a single plugin or aggregate options for all plugins? Is this a map where the plugin name will be the key?

I think this needs example.